Examples/source/Sequence/CustomInstrument.RSeries7822R/Code Modules/Imports/RSeries7822RDriverAPI/source/RSeriesDriver.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+
+namespace NationalInstruments.Example.CustomInstrument.RSeries7822DriverAPI
+{
+    /// <summary>
+    /// RSeries driver class which provides .NET APIs for Rseries card driver operations.
+    /// </summary>
+    public class RSeriesDriver
+    {
+        private ulong ReferenceID { get; }
+        private int Status { get; set; }
+
+        /// <summary>
+        /// Initializes Rseries card driver session.
+        /// </summary>
+        /// <param name="resourceName">InstrumentName.</param>
+        /// <exception cref="Exception"></exception>
+        public RSeriesDriver(string resourceName)
+        {
+            string bitFilePath = RSeriesCAPI.BitFilePath();
+            Status = RSeriesCAPI.OpenFPGA(resourceName, bitFilePath, out ulong fpgaRef);
+            ReferenceID = fpgaRef;
+            if (Status != 0)
+            {
+                throw new Exception($"Error in Open FPGA Ref, ErrorCode:{Status}, ResourceName:{resourceName}, Ref:{ReferenceID}, Bitfile:{bitFilePath}");
+            }
+        }
+
+        /// <summary>
+        /// Close Rseries card driver session.
```

Comment:
we need to change the RSeries in here to :
```suggestion
        /// Close RSeries card driver session.
```